content/blog/getting-started-with-the-hpe-data-services-cloud-console-public-rest-api.md

```diff
@@ -0,0 +1,182 @@
+
+### Versioning
+
+The major version number will be provided in the resource path as "v1" in this example:
+
+```jsoniq
+/api/v1/<resource group>/...
+```
+
+Here are some examples of these resource paths that contain several resource groups under the same root:
+
+```jsoniq
+/api/v1/storage-systems/...
+
+/api/v1/controllers/...
+
+/api/v1/volumes/...
+```
+
+Existing clients will be able to maintain the backward compatibility from the higher major version incremental, and adopt any newly introduced API. However, both the new and old version of the API will be supported until the announcement of the deprecation for the old version of the API. Nonetheless, the older major version will always be frozen with the exception of bug fixes. There will also be an announcement of deprecation in the header and sunset header.
+
+```jsoniq
+/api/v1/<resource group>/...
+
+/api/v2/<resource group>/...
+```
+
+### HTTP Request Methods
+
+| HTTP Verbs | Description                                                         |
+| ---------- | ------------------------------------------------------------------- |
+| GET        | Retrieves target resource                                           |
+| POST       | Creates an entity or changes state                                  |
+| PUT        | Replaces target resource with data part of the HTTP request payload |
+| DELETE     | Removes the resource                                                |
+
+### Authorization through OAuth2 Client Credential Workflow
+
+The client's application can issue a REST API request using the access token as the bearer of the token. The client can obtain this access token from the authorization API end point, after the client successfully authenticate through an associated customer's application credential (client-id and client-secret). This application credential is created by the DSCC user who has the permission to access resources (such as controllers, volumes etc.) under the DSCC instances. This access token expiration time, by default, is set for 7200 seconds (2 hours). When the resource server sees this expired access token, it returns 0x401 response (not authorized). The client must then reauthenticate using the associated client-id and client-secret to obtain the next access-token to use for the next REST API request.
+
+### Authorization Policies
+
+The client can only receive properties from the authorized API resources based on the Role Base Access Control for the user who created the client-credential pair (client-id and client-secret). This authorization derives from the organization, capability, and scope (roles) that the associated user is assigned. As the result, the authorization for the client application will inherit the user's permission who created the client-application registration under the API Gateway. Note that subsequent changes to the user's permission after the client application registered will impact the response returned based on current authority.
```

Comment:
***As the result, the authorization....who create the client application registration...*** (remove the "-" between client and application).

***...registration under the API Gateway.***

I believe this is the first time you mention the "API Gateway" in the blog.  Although you put pointers to your blog posts recently published on DEV portal, I would suggest to add a sentence about the API gateway here for the readers to get the context...if you see what I mean. Maybe something like that:

...registration under the API Gateway. The [API Gateway](https://developer.hpe.com/blog/api-console-for-data-services-cloud-console/) is used to register a client application and obtain the API account credentials (client-id and client-secret) for that client application. These credentials are required to generate short-lived access token that is used to make secure REST API calls to the DSCC application instance.

Reply:
This is definitely important, so I decided to create glossary of terms instead so that it's clear on what these are. Thanks!!